cpp/src/barretenberg/ecc/groups/affine_element.hpp

```diff
@@ -41,6 +43,31 @@ template <typename Fq, typename Fr, typename Params> class alignas(64) affine_el
               typename CompileTimeEnabled = std::enable_if_t<(BaseField::modulus >> 255) == uint256_t(0), void>>
     constexpr uint256_t compress() const noexcept;
 
+    /**
+     * @brief Get the limbs of the x and y coordinates of the group element.
+     *
+     * @return vector of the limbs of x, y coordinates (size 8)
+     */
+    std::vector<barretenberg::fr> get_coordinate_limbs() const
```

Comment:
@Rumata888 question: we might not end up using these methods in the immediate term in aztec3. do you think we should still keep these? Would they be useful anyway?